app/models/user.rb

```diff
@@ -1,5 +1,9 @@
 class User < ApplicationRecord
-  has_many :posts
-  has_many :likes
-  has_many :comments
+  has_many :comments, dependent: :destroy
+  has_many :posts, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  def recent_posts
+    posts.last(3)
+  end
```

Comment:
- [ ] I suggest that you make a DESC order you can do like `posts.order('created_at Desc').limit(3)`